test/scrabble_test.rb

```diff
@@ -5,9 +5,35 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def test_it_exists
+    scrabble = Scrabble.new("a")
+    assert_instance_of Scrabble, scrabble
+  end
```

Comment:
Pay careful attention to the interaction pattern and the tests given to you. If you build your tests out not matching the spec it will throw off all your functionality.